### PR TITLE
remove dead code: dart, bridge, cpp libraries...

### PR DESCRIPTION
**Pull request by peterjah**

### CODE REVIEW

**Review by Thykof — APPROVED** [review]

**Review by gregLibert — COMMENTED** [review]
Good cleaning !

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -56,7 +48,7 @@ jobs:
       - name: Update version
         run: echo -e "${GITHUB_REF_NAME}" > cmd/thyra-server/version.txt
       - name: Build binary for ${{ matrix.target }} on ${{ matrix.arch}}
-        run: GOOS=${{ matrix.target }} GOARCH=${{ matrix.arch }} CGO_ENABLED=1 CC=${{ matrix.cc }} CXX=${{ matrix.cxx }} go build -o thyra-server_${{ matrix.target }}_${{ matrix.arch }} ./cmd/thyra-server/
+        run: GOOS=${{ matrix.target }} GOARCH=${{ matrix.arch }} CGO_ENABLED=1 go build -o thyra-server_${{ matrix.target }}_${{ matrix.arch }} ./cmd/thyra-server/
```
[the remark this answers is not in the document]
Comment (peterjah) [reply .github/workflows/build.yml:51]: its still not clear to me what is CGO and when it is needed :/ so whats your advise about it?

[the remark this answers is not in the document]
Comment (gregLibert) [reply .github/workflows/build.yml:51]: I mostly used CGO when I needed to:
- import a c library
- generate a c-like library

I think that if we don't use fyne anymore, we can get rid of CGO.

This being said, we had weird issue with DNS resolver on MacOS that was solve using CGO (see [hello-world plugin](https://github.com/massalabs/thyra-plugin-hello-world/commit/372f933a9262e4bb0521dea989d025de1259d8dd)), even if apparently using golang 1.20 will [fix this](https://danp.net/posts/macos-dns-change-in-go-1-20/).

Which version og golang are we using ?

[the remark this answers is not in the document]
Comment (peterjah) [reply .github/workflows/build.yml:51]: right now we use 1.19.2

**Pull request merged**